pkg/ccl/importccl/import_stmt.go

```diff
@@ -2125,6 +2151,7 @@ func (r *importResumer) publishTables(ctx context.Context, execCfg *sql.Executor
 			}
 			newTableDesc.State = descpb.DescriptorState_PUBLIC
 			newTableDesc.OfflineReason = ""
+			newTableDesc.LastWriteableTime = hlc.Timestamp{}
```

Comment:
There exists a `newTableDesc.SetPublic()`, I think it makes sense to put `newTableDesc.LastWriteableTime = hlc.Timestamp{}` there and call that from here and below during revert?

Reply:
I was avoiding using the SetPublic and wrapper stuff so I could backport cleanly, and figured post-backport we could refactor on master